lib/pengine/pe_output.c

```diff
@@ -29,8 +30,7 @@ pe__name_and_nvpairs_xml(pcmk__output_t *out, bool is_list, const char *tag_name
         const char *param_name = va_arg(args, const char *);
         const char *param_value = va_arg(args, const char *);
         if (param_name && param_value) {
-            xmlSetProp(xml_node, (pcmkXmlStr) param_name
-                       , (pcmkXmlStr) (param_value ? param_value : ""));
+            xmlSetProp(xml_node, (pcmkXmlStr)param_name, (pcmkXmlStr)param_value);
```

Comment:
We already check that both param_name and param_value are not NULLs above. If one of the parameters is NULL, it simply won't be set. I use this feature in e.g. lib/pengine/native.c:809-813